src/util/processPool.ml

```diff
@@ -22,7 +22,7 @@ let run ~jobs ?(terminated=fun _ _ -> ()) tasks =
           Unix.open_process task.command
       in
       let pid = Unix.process_pid proc in
-      Catapult.Tracing.a_begin ~id:(string_of_int pid) "ProcessPool" ~args:[("command", `String task.command)];
+      Catapult.Tracing.a_begin ~id:(string_of_int pid) ~cat:[] "ProcessPool" ~args:[("command", `String task.command)];
```

Comment:
This is not directly related to OCaml 5, but required for the unlocked build to pass since a new version of catapult made this argument mandatory: https://github.com/ocaml/opam-repository/pull/22639.